### ISSUE TITLE
Add demo image

### ISSUE THREAD

**Issue by rowlandekemezie** [rowlandekemezie/contacto#1]

**Comment by ebukahills** [comment]
Need to add a gif demo image of how the CLI works, yes? I can send a PR

**Comment by rowlandekemezie** [comment]
![contacto-demo](https://cloud.githubusercontent.com/assets/15085641/26142419/515c66ae-3ad8-11e7-9ad1-c20c2a6f6187.gif)

**Comment by ebukahills** [comment]
We'd rather use this demo image?

**Comment by rowlandekemezie** [comment]
@flyg101, more like it because I've shipped same to the publisher. What do you think?

**Comment by ebukahills** [comment]
Yeah i did pretty much the same thing. Check the PR. I could redo it though

**Comment by rowlandekemezie** [comment]
You can borrow a leaf from mine. Make it a lot cleaner. I would also suggest, we don't see your username and stuff like that. Once we're fine with that, I can merge in the PR otherwise we'd maintain status quo.
